### PR TITLE
✨ Enable verbosity control

### PR DESCRIPTION
**Pull request by francostellari**
## Summary

Fix the verbosity arguments

## Related issue(s)

Fixes #

### CODE REVIEW

File: config/default/manager_auth_proxy_patch.yaml [hunk 1]
```diff
@@ -63,6 +63,7 @@ spec:
         - "--health-probe-bind-address=:8081"
         - "--metrics-bind-address=127.0.0.1:8080"
         - "--leader-elect"
+        - "--zap-log-leve={{.Values.verbosity | default 2}}"
```
[the remark this answers is not in the document]
Comment (francostellari) [reply config/default/manager_auth_proxy_patch.yaml]: @MikeSpreitzer fixed and enforced more checks on the value

File: config/default/manager_auth_proxy_patch.yaml [hunk 2]
```diff
@@ -63,6 +63,7 @@ spec:
         - "--health-probe-bind-address=:8081"
         - "--metrics-bind-address=127.0.0.1:8080"
         - "--leader-elect"
+        - "--zap-log-level={{max (.Values.verbosity | default 2 | int) 1}}"
```
Comment (MikeSpreitzer) [inline config/default/manager_auth_proxy_patch.yaml:66]: The defaulting here is new behvior, which should be documented in values.yaml and `docs/users.md`. Does not have to be done in this PR.

**Review by MikeSpreitzer — APPROVED** [review]
This is good enough to merge.

**Review by MikeSpreitzer — APPROVED** [review]
The two force-pushes since my last review did not change anything.
This is still good enough to merge.

**Review by pdettori — APPROVED** [review]
/lgtm

**Pull request merged**